src/utils/naming.js

```diff
@@ -0,0 +1,5 @@
+var createName = (role) => {
+    var number = Math.random().toFixed(3) * 1000;
+    return role + number.toString();
+}
+module.exports = createName;
```

Comment:
Please add an empty line at the end of each of these files, there are different file parsers that expect a file to end in a single empty line. All you would do to achieve this is to hit enter at the end of the last lines of all three of these files. Just a nitpick.